spec/README.mediawiki

```diff
@@ -1112,13 +1112,35 @@ This is by design and discourages certain types of spoofing.
 | orderid    || 16 || the order ID
 |}
 
-===Match notifications===
+===Match negotiation===
+
+Swap negotiation details will be relayed through the DEX with a series of
+notifications or progress reports.
+Both the DEX and the clients will need to serialize and sign the notification
```

Comment:
To be explicit that the notification sender is signing it (pretty obv though), this could say "Both ... will need to serialize and sign outgoing notification data."